src/main/java/com/deepoove/swagger/diff/compare/OperationsDiff.java

```diff
@@ -0,0 +1,139 @@
+      changedOperation = new ChangedOperation();
+      Operation oldOperation = oldOperations.get(method);
+      Operation newOperation = newOperations.get(method);
+      changedOperation.setSummary(newOperation.getSummary());
+
+      changedOperation.setVendorExtsFromGroup(extDiffer.diff(oldOperation, newOperation));
+
+      List<Parameter> oldParameters = oldOperation.getParameters();
+      List<Parameter> newParameters = newOperation.getParameters();
+      ParameterDiffResult parameterDiff = paramDiffer.diff(oldParameters, newParameters);
+      changedOperation.setAddParameters(parameterDiff.getIncreased());
+      changedOperation.setMissingParameters(parameterDiff.getMissing());
+      changedOperation.setChangedParameters(parameterDiff.getChanged());
+
+      if (!diffResult.hasContractChanges() && parameterDiff.hasOnlyCosmeticChanges()) {
+        diffResult.setHasOnlyCosmeticChanges(true);
+      }
+
+      for (ChangedParameter param : parameterDiff.getChanged()) {
+        ChangedExtensionGroup paramExtDiff = extDiffer.diff(param.getLeftParameter(), param.getRightParameter());
+        param.setVendorExtsFromGroup(paramExtDiff);
+        if (paramExtDiff.vendorExtensionsAreDiff()) {
+          diffResult.setHasContractChanges(true);
+          diffResult.setHasOnlyCosmeticChanges(false);
+        }
+      }
+
+      Property oldResponseProperty = getResponseProperty(oldOperation);
+      Property newResponseProperty = getResponseProperty(newOperation);
+      PropertyDiffResult propertyDiff = propertyDiffer.diff(oldResponseProperty, newResponseProperty);
+      changedOperation.setAddProps(propertyDiff.getIncreased());
+      changedOperation.setMissingProps(propertyDiff.getMissing());
+      changedOperation.setChangedProps(propertyDiff.getChanged());
+
+      changedOperation.setChangeResponseDescription(operationResponseHasCosmeticChanges(oldOperation.getResponses().values(), newOperation.getResponses().values()));
+      changedOperation.setChangeDescription(!Objects.equals(oldOperation.getDescription(), newOperation.getDescription()));
+      changedOperation.setChangeSummary(!Objects.equals(oldOperation.getSummary(), newOperation.getSummary()));
+      changedOperation.setChangeOperationId(!Objects.equals(oldOperation.getOperationId(), newOperation.getOperationId()));
+
+      ChangedExtensionGroup responseExtDiff = extDiffer.diffResGroup(oldOperation.getResponses(), newOperation.getResponses());
+      changedOperation.putSubGroup("responses", responseExtDiff);
+      if (responseExtDiff.vendorExtensionsAreDiff()) {
+        diffResult.setHasContractChanges(true);
+        diffResult.setHasOnlyCosmeticChanges(false);
+      }
+
+      if (changedOperation.isDiff()) {
+        diffResult.getChangedOperations().put(method, changedOperation);
+      }
+      if (!diffResult.hasContractChanges() && changedOperation.hasOnlyCosmeticChanges()) {
+        diffResult.setHasOnlyCosmeticChanges(true);
```

Comment:
I need both because if there are no changes, both flags would be false. So inverting one would not give the other. `isDiff` is only useful for that in `ChangedParameter`.